Add PiGlow overlay

lategoodbye opened the pull request:
The PiGlow is a small add-on board for the Raspberry Pi that provides
18 individually controllable LEDs (SN3218).

https://shop.pimoroni.com/products/piglow

pelwell commented:
Please fix the README formatting - no hard TABs, two blank lines between entries - and regenerate the defconfigs with `make ... savedefconfig`. Otherwise, it looks fine.

lategoodbye committed:
configs: enable LED driver for PiGlow

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>

lategoodbye committed:
overlays: Add PiGlow overlay

The PiGlow is a small add-on board for the Raspberry Pi that provides
18 individually controllable LEDs (SN3218) and uses the following pins:

P1 & P17 (3V3)
P2 (5V)
P3 (SDA)
P5 (SCL)
P14 (GND)

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>

lategoodbye commented:
I hope this is correct now

pelwell commented:
Perfect - thank you.
